ci/deploy.sh

```diff
@@ -24,13 +14,6 @@ else
 fi
 echo "Version: ${VERSION}"
 
-if [ "$1" == "api" ]
-then
-  (cd ./api && ./project.rb deploy-api --project all-of-us-workbench-test \
-     --account circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
-     --creds_file ~/gcloud-credentials.key --version $VERSION --promote)
-else
-  (cd ./ui && ./project.rb deploy-ui --project all-of-us-workbench-test \
-    --account circle-deploy-account@all-of-us-workbench-test.iam.gserviceaccount.com \
-    --version $VERSION --promote)
-fi
+(cd ./ui && ./project.rb deploy-ui --project all-of-us-workbench-test \
```

Comment:
This script used to deploy either API or UI depending on $1. Now it appears to always deploy UI. Did you intend to make this change?

Reply:
Never mind, discussed offline that this script no longer is used for deploying the API.

If we are going to keep this around for much longer, I'd rename it to deploy-ui.sh or whatever.

Reply:
I'm hoping I have established enough patterns now that anyone can do the same thing for UI.